Sources/LocalizedStringKit/LocalizedStringKit.m

```diff
@@ -14,57 +14,61 @@ @implementation LocalizedStringKit
 
 #pragma mark - Public
 
+NSString *_Nonnull primaryBundleName = @"LocalizedStringKit.bundle";
+
+NSURL *_Nullable alternateBundleSearchPath = nil;
```

Comment:
You're going to need to update these to match the changes to the `.h` file.

Reply:
Updated